ports/vhr/gameinfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gamelist>
+	<game>
+		<path>./Victory Heat Rally.sh</path>
+		<name>Victory Heat Rally</name>
+		<desc>Welcome to VHR! Unlock the secrets of speed in this turbocharged neo super scaler arcade racing experience. Dominate the track & drift your way to the finish line on the road to becoming the VHRC champion.</desc>
```

Comment:
```suggestion
		<desc>Welcome to VHR! Unlock the secrets of speed in this turbocharged neo super scaler arcade racing experience. Dominate the track &amp; drift your way to the finish line on the road to becoming the VHRC champion.</desc>
```
Not sure about this one since it's using utf-8 encoding.